widgets/bigquery-cdcTarget.json

```diff
@@ -33,6 +33,11 @@
     {
       "label" : "Advanced",
       "properties" : [
+        {
+          "name": "datasetName",
```

Comment:
Let's add this field info the the documentation as well i.e. `md` file.

Reply:
done :)